microcosm_postgres/encryption/registry.py

```diff
@@ -17,26 +17,35 @@
 )
 
 
-def parse_config(context_keys: Sequence[str],
-                 key_ids: Sequence[Union[str, Sequence[str]]],
-                 account_ids: Sequence[Union[str, Sequence[str]]],
-                 partitions: Sequence[Union[str, Sequence[str]]]) -> Mapping[str, Mapping[str, Sequence[str]]]:
-    return {
-        # NB: split key id on non-comma to avoid confusion with config parsing
-        context_key: {
+def parse_config(
```

Comment:
Just a question, why we don't store the config in the format returned by this function?

Reply:
This is some legacy stuff left over from the old encryption v1 days. We create a new structure that keys by the client id so it's easier to use later on.